db/seeds.rb

```diff
@@ -5,3 +5,22 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require "./db/data"
+
+local_goals = SeedData::GOALS
+
+local_goals.each do |goal|
+  goal_in_db = Goal.new()
```

Comment:
You don't need the parenthesis here (and I think it's more idiomatic Ruby to omit them)